packages/lighthouse-spa/src/app/leaderboard/pages/leaderboard/leaderboard.component.spec.ts

```diff
@@ -51,24 +53,20 @@ describe('LeaderboardComponent', () => {
   });
 
   it('should setup a leaderboard on component mount', () => {
-    expect(component.lighthouseLeaderboard).toEqual(
-      mockLeaderboard.rows as any // graphql type error: using general type of the whole object
-    );
+    expect(component.rows).toEqual(mockLeaderboardRows);
     expect(component.totalCount).toEqual(
       mockLeaderboardAPI.data.listLHLeaderboard.count
     );
     expect(component.isPageLoading).toBeFalsy();
   });
 
-  it('should change sort order of leaderboard', fakeAsync(() => {
-    component.handleToggleSortOption();
-    component.handleSortDirChange('DESC');
-    tick();
-    fixture.detectChanges();
-    expect(component.leaderboardSelectedSortOrder).toEqual('DESC');
-    expect(component.pageOffset).toEqual(0);
-    expect(component.isPageSortOptionOpen).toBeFalsy();
-  }));
+  // it('should change sort order of leaderboard', fakeAsync(() => {
+  //   component.handleSortClick({ column: '', sortDir: 'DESC' });
+  //   tick();
+  //   fixture.detectChanges();
+  //   expect(component.pageOffset).toEqual(0);
+  //   expect(component.isPageSortOptionOpen).toBeFalsy();
+  // }));
```

Comment:
If this test is invalid remove it. or uncomment it

Reply:
Done